src/NUnitEngine/nunit.engine/Runners/AbstractTestRunner.cs

```diff
@@ -236,9 +259,7 @@ public TestEngineResult Run(ITestEventListener listener, TestFilter filter)
         /// <param name="filter">A TestFilter used to select tests</param>
         public void StartRun(ITestEventListener listener, TestFilter filter)
         {
-            EnsurePackageIsLoaded();
-
-            RunTestsAsynchronously(listener, filter);
+            RunAsync(listener, filter);
```

Comment:
`StartRun` just wraps `RunAsync` now. Are we able to remove `StartRun` or should we keep it on the interface?